preCICE-Ubuntu-20.04/install-openfoam.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -ex
+
+USER="vagrant"
+
+# Add the signing key, add the repository, update:
+wget -q -O - https://dl.openfoam.com/add-debian-repo.sh | sudo bash
+
+# Install OpenFOAM v2006:
+sudo apt-get install -y openfoam2006-default
+openfoam-selector --set openfoam2006 --system
+# todo: Somehow the openfoam-selector is not enough - the binaries are discoverable, but the libraries not. Remedy:
+echo "source /etc/profile.d/openfoam-selector.sh" >> /home/vagrant/.bashrc
+
+# Get the OpenFOAM-preCICE adapter
+if [ ! -d "openfoam-adapter/" ]; then
+    sudo -u ${USER} git clone --branch master https://github.com/precice/openfoam-adapter.git
```

Comment:
Do we need the full repository?

Can we `wget -O - | tar zx ` an archive from github?
This is significantly faster.

Can we omit the history?
```suggestion
    sudo -u ${USER} git clone --depth=1 --branch master https://github.com/precice/openfoam-adapter.git
```

Reply:
Omiting the history is a good idea.

I still want the repository, as I would expect people to also do changes. I would like them to be able to contribute, even though then this becomes a question of target group.